components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/provider/SearchEngineSuggestionProvider.kt

```diff
@@ -24,6 +24,7 @@ import java.util.UUID
  * @property searchIcon the icon to be shown for the suggestion(s), same icon for all
  * @property maxSuggestions the maximum number of suggestions to be provided
  * @property charactersThreshold the minimum typed characters used to match to a search engine name
+ * @property suggestionsHeader optional parameter to specify if the suggestion should have a header
```

Comment:
This should have been there.  But since you're not touching anything in this file, can we please do this in a separate PR?  Thanks